tests/1d_dc/NonlocalPotentialBCWithSchottky.i

```diff
@@ -47,7 +47,7 @@ area = 5.02e-7 # Formerly 3.14e-6
 
         trans_ss_check = 1
         ss_check_tol = 1E-15
-        ss_tmin = 3*${relaxTime}
+        ss_tmin = 3e-9 # or 3*${relaxTime}
```

Comment:
We can actually wait on idaholab/moose#12633 to go in and then we can use the new input file arithmetic parsing capability to update this line

Reply:
Sounds good!